### PR TITLE
ADEN-4522 Update to latest version of prebid.js

### PR DESCRIPTION
**Pull request by dianafa**
https://wikia-inc.atlassian.net/browse/ADEN-4522

### CODE REVIEW

File: extensions/wikia/AdEngine/AdEngine2ContextService.class.php [hunk 1]
```diff
@@ -23,8 +25,7 @@ public function getContext( Title $title, $skinName ) {
 
 			$pageFairDetectionKey = AdEngine2Resource::getKey( 'wikia.ext.adengine.pf.detection' );
 			$pageFairDetectionUrl = ResourceLoader::makeCustomURL( $wg->Out, [ $pageFairDetectionKey ], 'scripts' );
-
-			$prebidBidderUrl = AssetsManager::getInstance()->getURL( 'prebid_prod_js', $type );
+			$prebidBidderUrl = AssetsManager::getInstance()->getURL( empty( $wgAllInOne ) ? 'prebid_debug_js' : 'prebid_prod_js' , $type );
```
Comment (bognix) [inline extensions/wikia/AdEngine/AdEngine2ContextService.class.php:28]: you already have `$wg` here - use it to keep it consistent.

**Review by bognix — APPROVED** [review]

File: extensions/wikia/AdEngine/AdEngine2ContextService.class.php (hunk 1 above)
Comment (dianafa) [reply extensions/wikia/AdEngine/AdEngine2ContextService.class.php:28]: `$wgAllInOne` and generally wg globals were proven to be faster than using `$wg->`

File: extensions/wikia/AdEngine/js/lookup/prebid/adaptersPricesTracker.js [hunk 2]
```diff
@@ -5,7 +5,8 @@ define('ext.wikia.adEngine.lookup.prebid.adaptersPricesTracker', [
 ], function (adaptersRegistry, prebid, log) {
 	'use strict';
 
-	var logGroup = 'ext.wikia.adEngine.lookup.prebid.adaptersPricesTracker';
+	var logGroup = 'ext.wikia.adEngine.lookup.prebid.adaptersPricesTracker',
+		validResponseStatusCode = 1;
```
Comment (bognix) [inline extensions/wikia/AdEngine/js/lookup/prebid/adaptersPricesTracker.js]: we already have errorCode defined in `ext.wikia.adEngine.lookup.prebid.adaptersPerformanceTracker`. I'm thinking about moving those status codes to `ext.wikia.adEngine.wrappers.prebid` - it's strictly related with prebid implementation and I think it makes sense to define it within prebid wrapper

File: extensions/wikia/AdEngine/js/lookup/prebid/adaptersPricesTracker.js [hunk 3]
```diff
@@ -19,19 +20,35 @@ define('ext.wikia.adEngine.lookup.prebid.adaptersPricesTracker', [
...
+	 * getStatusCode check is needed because even in case of error bid.pbAg value is 0.00
+	 * @param bid object
+	 * @returns {boolean}
+	 */
+	function isValidPrice(bid) {
+		var priceFromBidder = bid.pbAg;
+
+		return priceFromBidder !== '' &&
+			!isNaN(priceFromBidder) &&
+			typeof(priceFromBidder) !== 'boolean' &&
```
Comment (bognix) [inline extensions/wikia/AdEngine/js/lookup/prebid/adaptersPricesTracker.js]: `!isNaN` already takes care of that - you can remove this check

File: extensions/wikia/AdEngine/js/spec/lookup/prebid/adaptersPricesTracker.spec.js [hunk 4]
```diff
@@ -33,28 +33,80 @@ describe('ext.wikia.adEngine.lookup.prebid.adaptersPricesTracker', function () {
 		prebid: {
 			get: function () {
 				return {
-					getBidResponsesForAdUnitCode: mocks.getBidResponsesForAdUnitCode
+					getBidResponsesForAdUnitCode: mocks.getBidResponsesForAdUnitCode,
 				}
 			}
 		},
 		getBidResponsesForAdUnitCode: noop,
+		getStatusCode1: function() {
```
Comment (bognix) [inline extensions/wikia/AdEngine/js/spec/lookup/prebid/adaptersPricesTracker.spec.js]: `getValidStatusCode` ?

**Comment by bognix** [comment]
LGTM

**Comment by bognix** [comment]
LGTM, Travis failure not related.

**Pull request merged**